MPower/app/src/main/java/org/sagebionetworks/bridge/researchstack/MPowerOnboardingManager.java

```diff
@@ -0,0 +1,80 @@
+package org.sagebionetworks.bridge.researchstack;
+
+import android.content.Context;
+import android.content.Intent;
+
+import com.google.gson.GsonBuilder;
+
+import org.researchstack.backbone.model.survey.CustomSurveyItem;
+import org.researchstack.backbone.model.survey.SurveyItem;
+import org.researchstack.backbone.model.survey.SurveyItemAdapter;
+import org.researchstack.backbone.model.survey.factory.SurveyFactory;
+import org.researchstack.backbone.step.CustomStep;
+import org.researchstack.backbone.step.Step;
+import org.researchstack.backbone.task.NavigableOrderedTask;
+import org.researchstack.skin.onboarding.OnboardingManager;
+import org.researchstack.skin.ui.OnboardingTaskActivity;
+
+import java.util.List;
+
+/**
+ * Created by TheMDP on 1/12/17.
+ *
+ * This class is overridden to demonstrate to the developer how to convert
+ * custom JSON SurveyItem objects in "onboarding.json" into CustomSteps used
+ * in the onboarding process
+ */
+
+public class MPowerOnboardingManager extends OnboardingManager {
```

Comment:
@EricSiegNW this class should answer your question about implementing your own SurveyItems/Steps/Tasks/etc.